scout/constants/igv_tracks.py

```diff
@@ -48,6 +48,7 @@
     "indexURL": HG38REF_INDEX_URL,
     "cytobandURL": HG38CYTOBAND_URL,
     "aliasURL": HG38ALIAS_URL,
+    "chromosomeOrder": "chr1, chr2, chr3, chr4, chr5, chr6, chr7, chr8, chr9, chr10, chr11, chr12, chr13, chr14, chr15, chr16, chr17, chr18, chr19, chr20, chr21, chr22, chrX, chrY, chrM",
```

Comment:
This will work with chrMT, right?

Reply:
Of course, when I opened the file I could see that it's applied to the 38 only. Thanks!

Reply:
Ah, if you are asking if it will work for the mtDNA hg38 view for hg19 cases, it should be no change except that chrM moves over to the far right on the screen if anyone zooms out to "All" from the view.